index.js

```diff
@@ -181,9 +193,10 @@ class AbstractBtpPlugin extends EventEmitter {
           requestId: await _requestId(),
           data: { protocolData }
         })
-        this.emit('_connect')
+        this._emitConnect()
       })
 
+      this._ws.on('close', () => this._emitDisconnect())
```

Comment:
This line causes the integration tests to fail with `test.millie ... connector:app error Error: connection aborted`.